df_py/util/constants.py

```diff
@@ -40,7 +40,7 @@
     23295: ["0xe02a421dfc549336d47efee85699bd0a3da7d6ff"],
 }
 
-PREDICTOOR_MULTIPLIER = 1 - 1 / 1.201
+PREDICTOOR_MULTIPLIER = 0.201
```

Comment:
Your previous PR had also changed unit tests. Does this one not need changes to unit tests?